tests/integ/test_stored_procedure.py

```diff
@@ -1397,3 +1397,20 @@ def return_success(session_):
             )
             return
         raise
+
+
+def test_force_inline_code(session):
```

Comment:
Can this be tested in Snowpandas instead? This addition is only meaningful to Snowpandas code for now.

Reply:
we can, but that part code will not be covered in snowpark tests. I don't think it's a good practice to leave uncovered code here.

Reply:
sure, I can do (1), but I'm not sure about (2) whether it's a good idea to add a code link to a private repo right now.